.github/workflows/jetson-xavier-nx-devkit-emmc.yml

```diff
@@ -9,6 +25,11 @@ on:
         required: false
         type: boolean
         default: false
+      deploy-environment:
+        description: Environment to use for build and deploy
+        required: false
+        type: string
+        default: balena-staging.com
 
 permissions:
   id-token: write # This is required for requesting the JWT #https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#requesting-the-access-token
```

Comment:
I think we need to add:

```
actions: read
contents: read
packages: read
id-token: write # This is required for requesting the JWT #https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#requesting-the-access-token
pull-requests: write
```

e.g https://github.com/balena-os/balena-alliance-raspberrypi3/pull/473/commits/4af9184d60686b818190646ad21b9fc26b067a90

Reply:
To all of the workflow files

Reply:
thanks @rcooke-warwick , they seem to be present though on all 3 workflow files already